l10n_ch_scan_bvr/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Please have a look at template it has evolved a bit:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst